lib/api.js

```diff
@@ -577,6 +577,22 @@ api.declare({
 });
 
 /**
+ * List AMIs and their usage
+ */
+api.declare({
+  method: 'get',
+  route: '/internal/ami-usage',
+  name: 'amiUsage',
+  title: 'See the list of AMIs and their usage',
+  stability: API.stability.experimental,
+  scopes: [['ec2-manager:internals']],
```

Comment:
I am not very much familiar with the semantics of the service. Therefore I will leave review of scopes management to @jhford

Reply:
This is an endpoint that's only useful to the taskcluster team, and as a /internals/ endpoint, i'd like to keep it locked down like this.  This is the right scope.